README.md

```diff
@@ -0,0 +1,94 @@
+# Tribute Page
+
+> This is my first freecodecamp project on responsive design
+
+![screenshot](./img/arch.jpg)
```

Comment:
Can you fix this screenshot? I think it should include the webpage, not an image.

I would also recommend adding a more descriptive description. Let's try to talk about the technologies used.